Dockerfile

```diff
@@ -10,17 +10,13 @@ RUN git -C /home/opam/opam-repository pull origin master && opam update -uy
 # install non-OCaml dependencies
 COPY Makefile /home/opam/src/.
 RUN make depext
-RUN opam install dune=1.11.3
+RUN opam install dune=1.11.4
 
 #install pandoc
-WORKDIR /tmp
```

Comment:
Removing this workdir leads to a permission error, we should probably explicitly ask curl to write to `/tmp` with `-o /tmp/pandoc-2.1.3-1-amd64.deb`